docker-compose.yml

```
@@ -1,42 +1,86 @@
-version: "3.9"
+version: '3.9'
 
 services:
-    client:
-        container_name: prakticum-client
-        image: prakticum-client
-        build:
-            context: .
-            dockerfile: Dockerfile.client
-            args:
-              CLIENT_PORT: ${CLIENT_PORT}
-        restart: always
-        ports:
-            - "${CLIENT_PORT}:80"
-        environment:
-          - CLIENT_PORT=${CLIENT_PORT}
-          - SERVER_PORT=${SERVER_PORT}
-    server:
-        container_name: prakticum-server
-        image: prackicum-server
-        build:
-            context: .
-            dockerfile: Dockerfile.server
-            args:
-              SERVER_PORT: ${SERVER_PORT}
-        restart: always
-        ports:
-            - "${SERVER_PORT}:${SERVER_PORT}"
-        environment:
-          SERVER_PORT: ${SERVER_PORT}
+  client:
+    container_name: tanki-client
+    image: tanki-client
+    restart: always
+    command: yarn run start:client
+    build:
+      context: .
+      dockerfile: ./Dockerfile.client
+      args:
+        CLIENT_PORT: ${CLIENT_PORT}
+    ports:
+      - '${CLIENT_PORT}:${CLIENT_PORT}'
+    env_file:
+      - .env
+    volumes:
+      - ./:/app
+      - /app/node_modules
+    networks:
+      - tanki-net
+    depends_on:
+      - server
 
-    postgres:
-      image: postgres:14     
-      ports:
-        - "${POSTGRES_PORT}:${POSTGRES_PORT}"
-      environment:
-        POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
-        POSTGRES_USER: ${POSTGRES_USER}
-        POSTGRES_DB: ${POSTGRES_DB}
-      volumes:
-        - ./tmp/pgdata:/var/lib/postgresql/data
+  server:
+    container_name: tanki-server
+    image: tanki-server
+    restart: always
+    command: yarn run start:server
+    env_file:
+      - .env
+    build:
+      context: .
+      dockerfile: ./Dockerfile.server
+    ports:
+      - '${SERVER_PORT}:${SERVER_PORT}'
+    environment:
+      DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_USER}:${POSTGRES_PORT}/${POSTGRES_DB}
```

```suggestion
      DATABASE_URL: postgres://${POSTGRES_USER}:${POSTGRES_PASSWORD}@${POSTGRES_HOST}:${POSTGRES_PORT}/${POSTGRES_DB}
```
тут host, а не user https://www.prisma.io/dataguide/postgresql/short-guides/connection-uris#a-quick-overview